docs/architecture/entity-relations.md

```diff
@@ -0,0 +1,5 @@
+# Entity Relations
+
+Diagram below describes missing entity relations for Director.
```

Comment:
If we add `Entity relations` document, then it would be good to document all entity relations, not only missing ones.
Fortunately, there are no that much relations.